File: app/src/main/java/com/mobile/rxjava2andretrofit2/mine/model/base/IMineModel.java
package com.mobile.rxjava2andretrofit2.mine.model.base;

import java.util.Map;

import io.reactivex.Observable;
import okhttp3.ResponseBody;

/**
 * author    : xxxxxxxxxxx
 * e-mail    : xxxxxxxxxxx@qq.com
 * date      : 2019/3/10 16:57
 * introduce :
 */

public interface IMineModel {

    Observable<ResponseBody> mineData(Map<String, String> bodyParams);

    Observable<ResponseBody> submitFeedback(Map<String, String> bodyParams);

}

File: app/src/main/java/com/mobile/rxjava2andretrofit2/kotlin/mine/model/base/IMineModel.kt
package com.mobile.rxjava2andretrofit2.kotlin.mine.model.base

import io.reactivex.Observable
import okhttp3.ResponseBody

interface IMineModel {

    fun mineData(bodyParams: Map<String, String>): Observable<ResponseBody>

    fun mineDetails(bodyParams: Map<String, String>): Observable<ResponseBody>

}